website/content/docs/post-processors/index.mdx

```diff
@@ -12,3 +12,7 @@ Post-processors run after the image is built by the builder and provisioned by
 the provisioner(s). Post-processors are optional, and they can be used to
 upload artifacts, re-package, or more. For more information about
 post-processors, please choose an option from the sidebar.
+
+See [post-processor](/docs/templates/hcl_templates/blocks/build/post-processor) and
+[post-processors](/docs/templates/hcl_templates/blocks/build/post-processors)
```

Comment:
Do you think it'd be a good idea to do this :
```suggestion
See [`post-processor`](/docs/templates/hcl_templates/blocks/build/post-processor) and
[`post-processors`](/docs/templates/hcl_templates/blocks/build/post-processors)
```

So that we sort of emphase that this is code ? My hope is that it could put the brain in 'code mode' and so the s will be more visible.

Reply:
Nice! I think it makes sense! Let me change it for all of the components